README.md

```diff
@@ -4,23 +4,22 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
 This is the official operator to deploy and manage IBM block storage CSI driver.
 
 Supported container platforms:
-  - OpenShift v4.3
```

Comment:
@roysahar-ibm this file will be changed before merging this PR as part of https://github.com/IBM/ibm-block-csi-operator/pull/98